constructs/service-api-function.ts

```diff
@@ -91,7 +55,7 @@ export class ServiceApiFunction extends Construct {
 				':apigateway:',
 				cdk.Fn.ref('AWS::Region'),
 				':lambda:path/2015-03-31/functions/',
-				this.fn.functionArn,
+				this.functionArn,
```

Comment:
I didn't see you set this in the base class - is it created as a part of the super class of the "BaseLambdaHandler" class? the lambda.nodeJsfunction?

Reply:
Do you mean the `functionArn`? That's part of the base `NodejsFunction`, the built-in base function construct from AWS.